solr/modules/ltr/src/java/org/apache/solr/ltr/model/LTRScoringModel.java

```diff
@@ -293,6 +293,10 @@ public String getFeatureStoreName() {
    */
   public abstract float score(float[] modelFeatureValuesNormalized);
 
+  public float scoreWithMissingBranch(Float[] modelFeatureValuesNormalized) {
```

Comment:
Necessary in order not to force all models to implement missing branch variation